requirements.in

```diff
@@ -1,5 +1,5 @@
 numpy
-spacy==2.0.18
+spacy==2.1.3
```

Comment:
can we require `spacy>=2.1.3,<2.2` so that we don't pin users to a particular version?